platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -3257,9 +3260,7 @@ namespace platoon_strategic_ihp
                     // Note: Use current_lanlet list (which was determined based on vehicle pose) to find current lanelet ID. 
                     long current_lanelet_id = current_lanelets[0].second.id();
 
-                    // note: This is just mock info to compile the code. 
-                    // TODO: This should be updated according to arbitrary lane change requirements. 
-                    //int target_lanelet_id = current_lanelet_id;
+                    // note: This is just mock info to compile the code.                     
```

Comment:
No we are not assuming a left lanechange. I actually deleted those few lines since it was redundant.
I have used absolut value for crosstrack (line 3227) are you saying we shouldn't?

Reply:
Nope!  This all looks good now.